### PR TITLE
dynamic programming

### PR DESCRIPTION
**Pull request by sumitmohite0501**
I have added two efficient program 1) primitive calculator 2) placing parenthesis

### CODE REVIEW

File: DynamicProgramming/placing parenthesis [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class PlacingParentheses {
```
Comment (akashchandwani) [inline DynamicProgramming/placing parenthesis:3]: Please use proper file name for this file - PlacingParentheses.java

File: DynamicProgramming/placing parenthesis [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        } else {
+            assert false;
+            return 0;
+        }
+    }
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String exp = scanner.next();
+        System.out.println(getMaximValue(exp));
```
Comment (akashchandwani) [inline DynamicProgramming/placing parenthesis:25]: getMaximValue(exp) always returns 0

File: DynamicProgramming/placing parenthesis [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
```
Comment (akashchandwani) [inline DynamicProgramming/placing parenthesis:2]: Please provide description of what this algorithms solves.

File: DynamicProgramming/primitive_calculator [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import java.util.*;
+
+public class PrimitiveCalculator {
```
Comment (akashchandwani) [inline DynamicProgramming/primitive_calculator:3]: Above comments applicable to this file as well.

File: DynamicProgramming/placing parenthesis [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+public class PlacingParentheses {
+    private static long getMaximValue(String exp) {
+      //write your code here
+      return 0;
+    }
+
+    private static long eval(long a, long b, char op) {
```
Comment (akashchandwani) [inline DynamicProgramming/placing parenthesis:9]: How is this algorithm in Dynamic Programming?

**Review by akashchandwani — CHANGES_REQUESTED** [review]

File: DynamicProgramming/placing parenthesis (hunk 1 above)
Comment (sumitmohite0501) [reply DynamicProgramming/placing parenthesis]: sir i have commit 77116222b67ed4c5600586c93b58e2e5e7fc68e9 changes that you suggest

**Pull request closed without merging**